File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/SimpleTextWidget.java
package com.pkmnapps.activitydo.dataclasses;

public class SimpleTextWidget {
    String aid,uid;
    String head,body;

    public SimpleTextWidget(String uid, String aid, String head, String body) {
        this.aid = aid;//aid is of activity
        this.uid = uid;//uid is unique aid of this element
        this.head = head;
        this.body = body;
    }

    public String getAid() {
        return aid;
    }

    public void setAid(String aid) {
        this.aid = aid;
    }

    public String getHead() {
        return head;
    }

    public void setHead(String head) {
        this.head = head;
    }

    public String getBody() {
        return body;
    }

    public void setBody(String body) {
        this.body = body;
    }

    public String getUid() {
        return uid;
    }

    public void setUid(String uid) {
        this.uid = uid;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/SimpleTextWidget.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class SimpleTextWidget     //aid is of activity
//uid is unique aid of this element
(var uid: String?, var aid: String?, var head: String?, var body: String?) {
    fun getAid(): String? {
        return aid
    }

    fun setAid(aid: String?) {
        this.aid = aid
    }

    fun getHead(): String? {
        return head
    }

    fun setHead(head: String?) {
        this.head = head
    }

    fun getBody(): String? {
        return body
    }

    fun setBody(body: String?) {
        this.body = body
    }

    fun getUid(): String? {
        return uid
    }

    fun setUid(uid: String?) {
        this.uid = uid
    }
}